src/auth/userSession.ts

```diff
@@ -1,4 +1,4 @@
-
+import { MemoVoidIterator } from 'lodash'
```

Comment:
Looks like an accidental auto-import typo here

Reply:
This was a fix recommended by the linter to fix a problem.